web-wallet/src/style/dusk/language.css

```diff
@@ -53,26 +53,30 @@
   --control-small-font-size: 0.875em;
   --control-small-padding: 0.625em 0.875em;
 
-  --control-bg-color: #fff;
-  --control-bg-color-hover: var(--silver-metallic);
+  --non-button-control-bg-color: #fff;
+  --non-button-control-text-color: var(--smokey-black);
```

Comment:
I don't think it's worth it to add complexity to this `--control` variables by adding a `--non-button` case.

In our current design document even the selects have a smokey black background in light mode: the only "controls" keeping a white background are textboxes for now, if I'm not mistaken.

I think we should let @laremas think about it and then have a chat with him, but my guts tell me that we should probably get rid of these `--control` vars altogether.

Having said that: no need to change here in this PR, as this can be a temporary step while we formalize things.

An extra issue to keep track of the problem is needed anyway, though.

(cc @kieranhall @deuch13 )